config/application.rb

```diff
@@ -6,6 +6,14 @@
 
 module Loomio
   class Application < Rails::Application
+    # Load any ENV values from config/local_env.yml, if it exists
+    config.before_configuration do
```

Comment:
Yeah I'd like to avoid doing this if possible. `dotenv` should handle this for you with values in the `.env` file. Probably a thing we should put in the README, eh?